src/index.ts

```diff
@@ -89,13 +89,13 @@ const action = async (context: any): Promise<void> => {
       }
       case 'encoding': {
         context.copyToClipboard(`https://gfycat.com/${gfyname.toLowerCase()}`);
-        context.notify('gfycat is still processing the upload, but its URL has been copied to the clipboard anyway.');
+        context.notify('gfycat is still processing the upload, but its URL has been copied to the clipboard anyway');
         return;
       }
       default: throw new Error();
     }
   }
-  catch (err) {
+  catch {
     context.copyToClipboard(`https://api.gfycat.com/v1/gfycats/fetch/status/${gfyname}`);
     context.notify(`There was a problem processing the uploaded file. See the status at https://api.gfycat.com/v1/gfycats/fetch/status/${gfyname}`);
```

Comment:
Is there really no way to show the status directly instead of forcing the user to go to a website?

Reply:
I am learning today—after years—that `catch` statements don't require the `exception` argument. 🤦‍♂ Wow!

Reply:
It's a recent change, actually. https://mariusschulz.com/blog/optional-catch-binding-in-typescript

Reply:
I'll think more about this. It'd be nice to have a way to show the user something like a log of temporary, but persisting information which can include links. Like if the export progress window had a "details" view for each export that could be expanded and could be directly linked to from notifications. This would also address the "messaging" point at https://github.com/wulkano/kap/issues/797.

Currently, neither notifications nor the `context.setProgress()`  method offer a way to show more flexible status information.